lib/processCss.js

```diff
@@ -179,16 +179,8 @@ module.exports = function processCss(inputSource, inputMap, options, callback) {
 		parserPlugin(parserOptions)
 	]);
 
-	var postCssFrom;
-	if (options.query.legacySourceMaps !== false) {
-		// we need a prefix to avoid path rewriting of PostCSS
-		postCssFrom = "/css-loader!" + options.from;
-	} else {
-		postCssFrom = options.from;
-	}
-
 	pipeline.process(inputSource, {
-		from: postCssFrom,
+		from: options.from,
```

Comment:
Break maps in many cases

Reply:
Are you referring to the existing test suite? It is expected that it broke. I held off on updating the test suite as the comments in this review would determine how it would need to be updated.

The existing test suite is still using assumptions about source maps prior to the fix in this PR. I currently have the feedback I need and I intend to update the test suite prior to setting up the test repo.

If it isn't the test suite, is there a specific scenario you can describe that I can look into?